modules/inflowwind/src/InflowWind.f90

```diff
@@ -240,6 +240,20 @@ SUBROUTINE InflowWind_Init( InitInp,   InputGuess,    p, ContStates, DiscStates,
       END IF
 
          ! initialize sensor data:   
+         p%lidar%NumBeam = InputFileData%NumBeam
```

Comment:
I suggest putting this initialization in `Lidar_Init`. Technically all of these values should be passed in through `Lidar_Init`'s `InitInp` structure. Also, the validation logic in `InflowWind_ParseInputFileInfo` should be moved to `Lidar_Init` as well.

Reply:
I attempted this but it was not possible to be pass the variables through InitInp because (as far as I can tell) these cannot be specified from the InflowWind input file.